bolt/lib/Passes/PatchEntries.cpp

```diff
@@ -98,6 +98,17 @@ void PatchEntries::runOnFunctions(BinaryContext &BC) {
     });
 
     if (!Success) {
+      // We can't change output layout for AArch64 due to LongJmp pass
```

Comment:
Is it safe to just continue without ignoring the function? 
And in general, what's the interaction between PatchEntries and LongJmp passes? Can we patch the entry and rely on LongJmp to insert trampolines?

Reply:
I would say yes. Internally we're using rewrite option so old text is completely removed, so we expect everything to be disassembled properly. To be honest I didn't even know that patch entries is forced to run. Also patch entries skips functions which size is smaller than 128 anyway, that we might consider not safe too, but we're doing it :) And finally the chance to call old function on ARM is smaller due to +- 128 mb limit.